docs/recommendations/DNA/alleles.md

```diff
@@ -6,37 +6,9 @@ Allele: a series of variants on one chromosome.
 
 ## Syntax
 
-<table class="syntax">
```

Comment:
The loss of these examples is still unintended, right?

Reply:
Yes, for this round. Fixed.